composer.json

```diff
@@ -25,15 +25,17 @@
     "require": {
         "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "composer-plugin-api": "^1.0 || ^2.0",
-        "laminas/laminas-component-installer": "^1.0 || ^2.1.2"
+        "laminas/laminas-component-installer": "^1.0 || ^2.1.2",
+        "vimeo/psalm": "^4.13"
```

Comment:
@Ocramius should `vimeo/psalm` be in `require-dev` ? Looking at other package, it is in require-dev https://github.com/laminas/laminas-code/blob/c99ef8e5629c33bfaa3a8f1df773e916af564cd6/composer.json#L21

Reply:
@Ocramius I moved `vimeo/psalm` to `require-dev` https://github.com/laminas/laminas-skeleton-installer/pull/24/commits/c17ea32fafc1d5381591b075fdc5428d16ce9c99